content/doc/installation/run-as-service-in-ubuntu.en-us.md

```diff
@@ -0,0 +1,97 @@
+---
+date: "2017-07-21T12:00:00+02:00"
+title: "Run as service in Linux"
+slug: "linux-service"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "installation"
+    name: "Linux service"
+    weight: 20
+    identifier: "linux-service"
+---
+
+### Run as service in Ubuntu 16.04 LTS  
+ 
+#### Using supervisor  
+
+Install supervisor by running below command in terminal:  
+```sudo apt install supervisor```  
+
+Create a log dir for the supervisor logs(assuming gitea is installed in /home/git/gitea/):  
+```mkdir /home/git/gitea/log/supervisor```  
+
+Open supervisor config file in vi/vim/nano etc.  
+```sudo vim /etc/supervisor/supervisord.conf```  
+
+And append the following code at the end of the file, [reference](https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea):  
+```
```

Comment:
Just link to the contrib-file: https://github.com/go-gitea/gitea/blob/master/contrib/supervisor/gitea